lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequest.java

```diff
@@ -86,6 +103,7 @@ public int getNumberOfCores() {
    * @return the {@link org.apache.reef.driver.catalog.NodeDescriptor} used as the template for this
    * {@link EvaluatorRequest}.
    */
+  @Deprecated
```

Comment:
Please add javadoc `@deprecated` as well and document when it was deprecated and what to use instead.